osi_payment_method/README.rst

```diff
@@ -0,0 +1,79 @@
+==============
+Payment Method
+==============
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-ursais%2Fosi--addons-lightgray.png?logo=github
+    :target: https://github.com/ursais/osi-addons/tree/12.0/sale_subscription_brand
+    :alt: ursais/osi-addons
+
+|badge1| |badge2| |badge3|
+
+This module adds a payment method on the partners and set it by default on the
+sales orders, purchase orders, customer invoices and vendor bills.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+* Go to Contacts
+* Create or select a partner
+* Go to the "Sales & Purchase" tab
```

Comment:
It looks like this field is now in the 'Accounting' tab so this should be updated.